nautobot/core/authentication.py

```diff
@@ -121,70 +121,6 @@ def has_perm(self, *args, **kwargs):
         return False
 
 
-class LDAPBackend:
-    """
-    Wrapper for stock `django-auth-ldap` LDAP authentication.
-
-    This backend that validates usability based on whether the plugin is
-    installed and configured, otherwise it will return a dummy backend to allow
-    authentication to proceed using other configured backends.
-    """
-
-    is_usable = False
-
-    def __new__(cls, *args, **kwargs):
-        try:
-            from django_auth_ldap.backend import (
-                LDAPBackend as LDAPBackend_,
-                LDAPSettings,
-            )
-            import ldap
-        except ModuleNotFoundError as e:
-            if getattr(e, "name") == "django_auth_ldap":
-                logging.error("LDAP authentication has been configured, but django-auth-ldap is not installed.")
-
-        # Try to import `ldap_config.py`
-        # FIXME(jathan): Have this read from `django.conf.settings` instead vs.
-        # another config file that has to be dropped inside of the Nautobot code
-        # deployment.
-        try:
-            from nautobot.core import ldap_config
```

Comment:
Can also remove `nautobot/core/ldap_config.py` from `.gitignore` now, I think?

Reply:
Ah cool I'll get that in the next one.